shell/common/node_bindings.cc

```diff
@@ -324,6 +324,7 @@ bool IsAllowedOption(const std::string_view option) {
 
   // This should be aligned with what's possible to set via the process object.
   static constexpr auto options = base::MakeFixedFlatSet<std::string_view>({
+      "--diagnostic-dir",
```

Comment:
Should document this https://github.com/electron/electron/blob/main/docs/api/command-line-switches.md#nodejs-flags